sunpy/net/hek/tests/test_hek.py

```diff
@@ -220,6 +238,37 @@ def test_query_multiple_operators():
                             attrs.hek.OBS.Observatory == "GOES")
     assert len(results) == 7
 
+@pytest.mark.remote_data
+def test_astropy_unit_parsing(read_unit_attributes, read_coord_attributes):
+    client = hek.HEKClient()
```

Comment:
In other test files, we create one client fixture and pass that into the unit tests. Can we do that here as well please?

Reply:
Ok